lib/shex/CHCS_clinical_chemistry.shex

```diff
@@ -0,0 +1,31 @@
+PREFIX chcs: <http://hokukahu.com/schema/chcss#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX map: <http://shex.io/extensions/Map/#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX fm: <http://hokukahu.com/map/>
+BASE <http://hokukahu.com/shapes/>
+
+start = <labs>
+
+<labs> {                                                                                 # <63-1003681>
+  ^chcs:clinical_chemistry-63 {                                                           # <63_04-6929184.919299_1003681>
+    a [chcs:Lab_Result-63],
+  # chcs:identifier                     LITERAL  %map:{ fm:order.identifier %},          # "63-000007"
+    rdfs:label                          LITERAL,                                         # "DOEXXX,JANE"
+    chcs:patient-63 {
+      chcs:identifier                   LITERAL  %map:{ fm:subject.identifier %} },      # <6-7893413> "DOEXXX,JANE"
+  },
+  a [chcs:Clinical_Chemistry-63_04],
+  chcs:identifier                       LITERAL  %map:{ fm:order.identifier %},          # "63-000007"
+  chcs:login_date_time-63_04            LITERAL  %map:{ fm:apparentDate %},              # "1990-01-01T08:07:01Z"

```

Comment:
eric and glenna, I'd like all timestamps to be 00:00:00 .  I see a couple that are not, in https://github.com/rdf-pipeline/noflo-rdf-components/blob/montreal-demo/lib/shex/CHCS_clinical_chemistry.shex Would one of you please change them and then delete the file history from github?  thanks